pkg/server/errors_test.go

```diff
@@ -23,20 +23,20 @@ import (
 
 	"github.com/codenotary/immudb/embedded/store"
 	immuerrors "github.com/codenotary/immudb/pkg/errors"
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestMapServerError(t *testing.T) {
 	err := mapServerError(store.ErrIllegalState)
-	assert.Equal(t, ErrIllegalState, err)
+	require.Equal(t, ErrIllegalState, err)
 
 	err = mapServerError(store.ErrIllegalArguments)
-	assert.Equal(t, ErrIllegalArguments, err)
+	require.Equal(t, ErrIllegalArguments, err)
```

Comment:
it may be better to improve error comparison with `require.ErrorIs(t, err, ErrIllegalArguments)`